src/main/java/lotto/domain/ShuffledNumbers.java

```diff
@@ -0,0 +1,28 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ShuffledNumbers {
+	private static final int RANGE_FROM = 1;
+	private static final int RANGE_TO = 46;
+	private static final int MINIMUM_SIZE = 0;
+	private static final int MAXIMUM_SIZE = 45;
+	private final List<Number> numbers;
+
+	public ShuffledNumbers() {
+		numbers = IntStream.range(RANGE_FROM, RANGE_TO)
+				.mapToObj(Number::from)
+				.collect(Collectors.toList());
+		Collections.shuffle(numbers);
+	}
```

Comment:
```suggestion
	private static final int RANGE_FROM = 1;
	private static final int RANGE_TO = 45;
	private static final int MINIMUM_SIZE = 0;
	private static final int MAXIMUM_SIZE = 45;
	private final List<Number> numbers;

	public ShuffledNumbers() {
		numbers = IntStream.rangeClosed(RANGE_FROM, RANGE_TO)
				.mapToObj(Number::from)
				.collect(Collectors.toList());
		Collections.shuffle(numbers);
	}
```
이렇게 변경해볼 수도 있겠네요 더불어 RANGE_TO 보다 조금 더 로또와 관련된 상수명으로 변경 가능할거같아요

Reply:
더 명확한 표현이 가능하겠네요!
감사합니다🙏